src/style.css

```
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header-content {
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+li {
+  display: inline-block;
+  margin: 0 15px;
+}
+
+.nav__item {
```

first child shouldn't have margin
```suggestion
.nav__item:not(:first-child) {
```

---

Thanks, I've updated it 
10382f7918d9effcd44b7a5de593421554c60af2